samples/csharp_dotnetcore/03.welcome-user/readme.md

```diff
@@ -13,8 +13,55 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
 - Hit F5
 ## Visual Studio Code
 - Open `botbuilder-samples/samples/csharp_dotnetcore/03.welcome-user` folder
-- Bring up a terminal, navigate to botbuilder-samples/samples/csharp_dotnetcore/03.welcome-user
-- Type 'dotnet run'.
+- Bring up a terminal, navigate to `botbuilder-samples/samples/csharp_dotnetcore/03.welcome-user`
+- Type `dotnet run`.
+
+## Update packages
+
+- In Visual Studio right click on the solution and select "Restore NuGet Packages".
+- In Visual Studio Code type `dotnet restore`
+
+# Use Azure Blob Storage or Azure CosmosDB
+
+##### In Visual Studio

```

Comment:
This section can be removed. It's not really relevant for this sample and seems to make it more complex.

Reply:
We removed the `Azure Blob Storage` section from the Readme.